README.md

```diff
@@ -57,6 +57,10 @@ JSON file. You may also set the GOOGLE_APPLICATION_CREDENTIALS environment varia
     ./cloud_sql_proxy -dir=/cloudsql -instances=my-project:us-central1:sql-inst=tcp:3306 &
     mysql -u root -h 127.0.0.1
 
+    # For specifying another host:
+    ./cloud_sql_proxy -dir=/cloudsql -instances=my-project:us-central1:sql-inst=tcp:0.0.0.0:3306 &
+    mysql -u root -h 0.0.0.0
```

Comment:
`mysql -u root -h 0.0.0.0` works in this container: https://hub.docker.com/r/syoya/cloudsql-proxy/~/dockerfile/
And this container can be accessed from another container (like application container). When I specified `=tcp:127.0.0.1:3306`, the container wasn't accessible.

But this is hard to understand, so I will rewrite this line `mysql -u root -h [machine-ip]`.